.gitignore

```diff
@@ -1,7 +1,7 @@
 .DS_Store
 *~
 *.swp
-build*
+build
```

Comment:
Just a suggestion:

I would keep `build*` for ignoring all `build`, `build_debug`, `build_release`, `build_test` ...

If you don't want to call the script in another way (e.g. `configure_build.sh`, `generate_build.sh`), you may add a specific rule in `.gitignore` (i.e. `!build.sh`)

Reply:
Since the script performs just the cmake configuration and does not build anything I would call it `configure.sh`.

Reply:
I placed a make call in the script and preserved the current name. I also added a `!build.sh` rule in `.gitignore` and restored `build*`